terraform/modules/concourse_worker/worker_config.tf

```diff
@@ -102,7 +103,7 @@ write_files:
     content: ${base64encode(local.healthcheck_file)}
     owner: root:root
     path: /home/root/healthcheck.sh
-    permissions: '0700' 
+    permissions: '0700'
```

Comment:
I don't see what's changed here

Reply:
The auto `fmt` just removed the trailing space